lib/update_lists_from_pardot.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateListsFromPardot
+
+  def self.call
+    new.start_update
+  end
+
+  def start_update
+    lists = Pardot.client.lists.query(is_greater_than: 0)
+    lists['list'].each do |list|
+      next unless list['is_public'] == 'true'
+
+      @pardot_list = List.where(pardot_id: list['id']).first_or_create do |plist|
+        plist.title = list['title']
+        plist.description = list['description']
+      end
```

Comment:
you already have some of the stuff we were talking about

Reply:
expanded to remove lists that are deleted from Pardot

Reply:
also this confirms that the similar code in the SyncPardot... file has some confusing naming, as this uses `list['id']` where the other same code uses `subscription['id']`